tests/test_data.py

```diff
@@ -1011,8 +1011,21 @@ def test_tabulate_behavior_data_tracking_task(self):
         plt.xlabel('Reward trial #'); plt.ylabel('Time (sec)')
         plt.ylim(15,25); plt.legend()
         figname = 'tabulate_tracking_trial_segment_lengths_churro.png'
-        visualization.savefig(write_dir, figname)       
+        visualization.savefig(write_dir, figname)   
 
+    def test_tabulate_random(self):
+
+        subjects = ['Leo', 'Leo']
+        ids = [1957, 1959]
+        dates = ['2025-02-13', '2025-02-13']
+
+        df = wrapper_tabulate_random(data_dir, subjects, ids, dates, metadata = ['sequence_params'])
+        self.assertEqual(len(df), 66) #check correct length 
+        self.assertEqual(len(df.columns), 10+1)  #check correct number of columns
+        for loc in df['target_location']:
+            self.assertEqual(loc.shape[0], 3)
+            self.assertLess(np.linalg.norm(loc), 10)
+        
```

Comment:
it would be nice to save a plot of the targets and trajectories from when the targets appear until they are acquired